Views/Home/Index.cshtml

```diff
@@ -111,91 +114,91 @@
-            $('#elementTable').on('click', '.delete-button', function () {
-                elementIdToDelete = $(this).data('id');
-                rowToDelete = $('#elementTable').DataTable().row($(this).closest('tr'));
-            });
-
-            $('#confirmDeleteButton').on('click', function () {
-
-                if (elementIdToDelete) {
-                    $.ajax({
-                        url: 'Home/Delete/' + elementIdToDelete, // API URL
-                        type: 'DELETE',
-                        success: function (data) {
-                            console.log(data);
-                            if (data.success) {
-                                // Delete the line in DataTables
-                                rowToDelete.remove().draw();
-                                console.log('Element successfully deleted.');
-                                // Close the modal
-                                $("[data-dismiss=modal]").trigger({ type: "click" });
-                            } else {
-                                console.error('Element not found or deletion failed.');
-                            }
-                        },
-                        error: function (xhr, status, error) {
-                            console.error('Error deleting element: ' + error);
-                        }
-                    });
-                }
-            });
+            // var elementIdToDelete;
+            // var rowToDelete; // Select the parent line of this button
+
+            // $('#elementTable').on('click', '.delete-button', function () {
+            //     elementIdToDelete = $(this).data('id');
+            //     rowToDelete = $('#elementTable').DataTable().row($(this).closest('tr'));
+            // });
+
+            // $('#confirmDeleteButton').on('click', function () {
+
+            //     if (elementIdToDelete) {
+            //         $.ajax({
+            //             url: 'Home/Delete/' + elementIdToDelete, // API URL
+            //             type: 'DELETE',
+            //             success: function (data) {
+            //                 console.log(data);
+            //                 if (data.success) {
+            //                     // Delete the line in DataTables
+            //                     rowToDelete.remove().draw();
+            //                     console.log('Element successfully deleted.');
+            //                     // Close the modal
+            //                     $("[data-dismiss=modal]").trigger({ type: "click" });
+            //                 } else {
+            //                     console.error('Element not found or deletion failed.');
+            //                 }
+            //             },
+            //             error: function (xhr, status, error) {
+            //                 console.error('Error deleting element: ' + error);
+            //             }
+            //         });
+            //     }
```

Comment:
This is the Javascript function for deleting a project. l've commented on it so that l can reuse it later if need be, but l can remove it because it's not currently useful. the delete function has been implemented differently. It was just to save that.

Reply:
Ok, maybe a comment explaining this at the start of the commented-out code would be useful?

Reply:
I've already deleted it completely as this function is not useful at the moment